plugin/plugin_settings.py

```diff
@@ -117,7 +117,7 @@ def __update_settings_from_project_if_needed(self):
         self.__load_vars_from_settings(settings_handle, Settings.PREFIXES)
         log.debug(" All overrides applied.")
 
-    def __load_vars_from_settings(self, settings_handle, prefixes=[""]):
+    def __load_vars_from_settings(self, settings_handle, prefixes=None):
```

Comment:
I also like the first one more. But Codacy doesn't - https://www.codacy.com/app/zabugr/EasyClangComplete/file/4318883104/issues/source?bid=3285178&fileBranchId=3285178#l120